CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # ChangeLog
 
+## v0.3.2
+
+SCIP schema:
+
+- Added new `SymbolRole`: `ForwardDefinition`
+  (https://github.com/sourcegraph/scip/pull/217)
+
```

Comment:
```suggestion

scip CLI:

- `scip print` supports turning off colorful output using `--color=false` or
   by setting the environment variable `NO_COLOR=1`.
   (https://github.com/sourcegraph/scip/pull/214)
```

Reply:
Woops. I'll add it in separate PR, okay?